apps/server/src/schema/user/user-types.ts

```diff
@@ -102,6 +103,7 @@ export const UserDto = builder.prismaObject('User', {
     currentOrganization: t.relation('currentOrganization', { nullable: true, type: OrganizationDto }),
     comments: t.relation('comments', { nullable: false }),
     taggedInComment: t.relation('taggedInComment', { nullable: false }),
+    preferences: t.relation('preferences', { nullable: true, type: PreferencesDto }),
```

Comment:
```suggestion
    preferences: t.relation('preferences', { nullable: true }),
```
pothos is smart and knows